pointcloud/basic_octree_utils_test.go

```diff
@@ -327,3 +330,63 @@ func printBasicOctree(logger golog.Logger, bOct *BasicOctree, s string) {
 		printBasicOctree(logger, v, s+"-+-")
 	}
 }
+
+// Test the functionalities involved with converting a pointcloud into a basic octree.
+func TestBasicOctreeCollision(t *testing.T) {
+	startPC, err := makeFullPointCloudFromArtifact(
+		t,
+		"slam/example_cartographer_outputs/viam-office-02-22-1/pointcloud/pointcloud_0.pcd",
+		BasicType,
+	)
+	test.That(t, err, test.ShouldBeNil)
+
+	center := getCenterFromPcMetaData(startPC.MetaData())
+	maxSideLength := getMaxSideLengthFromPcMetaData(startPC.MetaData())
+
+	basicOct, err := NewBasicOctree(center, maxSideLength)
+	test.That(t, err, test.ShouldBeNil)
+
+	startPC.Iterate(0, 0, func(p r3.Vector, d Data) bool {
+		_, _, blueProb := d.RGB255()
+		d.SetValue(int(blueProb))
```

Comment:
[nit] add comment as to why we are adding the blue value. Example: "Blue channel is used to determine probability in pcds produced by cartographer

Reply:
Done